src/plugins/arkeo/StakingCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@chakra-ui/react'
+import { DefiProvider } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import type { EarnOpportunityType } from 'features/defi/helpers/normalizeOpportunity'
+import { useTranslate } from 'react-polyglot'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { ArkeoCard } from './ArkeoCard'
+
+type StakingCardProps = {
+  onClick: (arg: EarnOpportunityType) => void
+} & EarnOpportunityType
```

Comment:
Again not that big of a deal given this is a one-time use thing but we should avoid passing whole structures around when we can pass ids, that would also avoid having to explicitly destructure `onClick` and `...rest`